test/actions/katello/organization_test.rb

```diff
@@ -130,6 +130,23 @@ class ManifestRefreshTest < TestBase
                                  dependency: found.first.output
                                 )
     end
+
+    it 'plans & create audit record for organization with comment manifest refreshed' do
+      acme_org = get_organization(:empty_organization)
+      rhel7 = katello_repositories(:rhel_7_x86_64)
+      acme_org.stubs(:owner_details).returns({})
+      acme_org.products.stubs(:redhat).returns([rhel7.product])
+      action.stubs(:rand).returns('1234')
+
+      action.stubs(:link!)
+      action.stubs(:lock!)
+      action.stubs(:exclusive_lock!)
+
+      plan_action(action, acme_org)
+      assert_difference 'Audit.count', 1 do
```

Comment:
Note - when I have added `acme_org.audits`, it was giving  `<ActiveRecord::Associations::CollectionProxy []>` whereas in `Audit`  getting a new record for Organization.